docs/faq.md

```diff
@@ -2,119 +2,160 @@
 
 ## Should I move shortcodes to content blocks?
 We think so. Blocks are designed to be visually representative of the final look, and they will likely become the expected way in which users will discover and insert content in WordPress.
 
 ## Will Gutenberg be made properly accessible?
-Accessibility is not an afterthought. Although not every aspect of Gutenberg is accessible at the moment, we are tracking issues in https://github.com/WordPress/gutenberg/labels/Accessibility, and we are fixing them. We understand that WordPress is for everyone, and that accessibility is about inclusion. This is a key value for us.
 
-If you would like to contribute to the accessibility of Gutenberg, we could always use people to test, and contribute.
+Accessibility is not an afterthought. Not every aspect of Gutenberg is accessible at the moment, but we are tracking issues in https://github.com/WordPress/gutenberg/labels/Accessibility, and we are fixing them. We understand that WordPress is for everyone, and that accessibility is about inclusion. This is a key value for us.
+
+If you would like to contribute to the accessibility of Gutenberg, we can always use more people to test and contribute.
 
 ## Are there any design resources for Gutenberg?
+
 Yes, primarily https://github.com/WordPress/gutenberg/blob/master/docs/design.md
 
 We are still adding more documentation.
 
 ## How is data stored? I've seen HTML comments, what is their purpose?
-Our approach — as outlined in <a href="https://make.wordpress.org/core/2017/01/17/editor-technical-overview/">the technical overview introduction</a> — was to augment the existing data format with the notion and intention of blocks in a way that didn't break the decade and a half fabric of content that WordPress has provided. In other terms, this optimizes for a format that prioritizes human readability (the html document of the web) and easy-to-render-anywhere over a machine convenient file (JSON in post-meta) that benefits the editing context primarily.
+
+Our approach—as outlined in <a href="https://make.wordpress.org/core/2017/01/17/editor-technical-overview/">the technical overview introduction</a>—is to augment the existing data format in a way that doesn’t break the decade-and-a-half-fabric of content WordPress provides. In other terms, this optimizes for a format that prioritizes human readability (the HTML document of the web) and easy-to-render-anywhere over a machine convenient file (JSON in post-meta) that benefits the editing context primarily.
 
 This also <a href="https://github.com/WordPress/gutenberg/issues/1516">gives us the flexibility</a> to store those blocks that are inherently separate from the content stream (reusable pieces like widgets or small post type elements) elsewhere, and just keep token references for their placement.
 
+We suggest you look at the [language of Gutenberg](/language/) to learn more about how this aspect of the project works.
```

Comment:
Would a relative link `../language/` be better supported for if/when these docs are moved further down a subdirectory?